app/parse.py

```diff
@@ -15,9 +21,83 @@ class Product:
     num_of_reviews: int
 
 
-def get_all_products() -> None:
-    pass
+driver = webdriver.Chrome()
```

Comment:
Consider using a context manager or a try-finally block to ensure that the WebDriver is properly closed even if an exception occurs during execution. This will help prevent leaving browser instances open.